lib/optimize/SplitChunksPlugin.js

```diff
@@ -90,45 +90,52 @@ module.exports = class SplitChunksPlugin {
 			minChunks: options.minChunks || 1,
 			maxAsyncRequests: options.maxAsyncRequests || 1,
 			maxInitialRequests: options.maxInitialRequests || 1,
-			getName: SplitChunksPlugin.normalizeName(options.name) || (() => {}),
+			getName:
+				SplitChunksPlugin.normalizeName({
+					name: options.name,
+					automaticNameDelimiter: options.automaticNameDelimiter
+				}) || (() => {}),
 			filename: options.filename || undefined,
-			getCacheGroups: SplitChunksPlugin.normalizeCacheGroups(
-				options.cacheGroups
-			)
+			getCacheGroups: SplitChunksPlugin.normalizeCacheGroups({
+				cacheGroups: options.cacheGroups,
+				automaticNameDelimiter: options.automaticNameDelimiter
+			})
 		};
 	}
 
-	static normalizeName(option) {
-		if (option === true) {
+	static normalizeName({ name, automaticNameDelimiter }) {
```

Comment:
Just `nameDelimiter` ?  Would be even cleaner :) (nitpick)

Reply:
I would stay with automaticNameDelimiter. This make it obvious that this is only used when `name` is in `automatic` mode and not in general.